test/import/FSHImporter.Invariant.test.ts

```diff
@@ -203,5 +158,83 @@ describe('FSHImporter', () => {
         expect(loggerSpy.getLastMessage('error')).toMatch(/File: File2\.fsh.*Line: 2 - 4\D*/s);
       });
     });
+
+    describe('#assignmentRule', () => {
+      it('should parse an invariant with assigned value rules', () => {
+        const input = `
+        Invariant: rules-1
+        Severity: #error
+        Description: "This has some rules."
+        * requirements = "This invariant exists because I willed it so."
+        * expression = "name.exists()"
+        `;
+
+        const result = importSingleText(input, 'InvariantRules.fsh');
+        expect(result.invariants.size).toBe(1);
+        const invariant = result.invariants.get('rules-1');
+        expect(invariant.name).toBe('rules-1');
+        const severityCode = new FshCode('error')
+          .withLocation([3, 19, 3, 24])
+          .withFile('InvariantRules.fsh');
+        expect(invariant.severity).toEqual(severityCode);
+        expect(invariant.description).toBe('This has some rules.');
+        expect(invariant.rules.length).toBe(2);
+        assertAssignmentRule(
+          invariant.rules[0],
+          'requirements',
+          'This invariant exists because I willed it so.'
+        );
+        assertAssignmentRule(invariant.rules[1], 'expression', 'name.exists()');
+      });
+
+      it('should parse an instance with assigned values that are an alias', () => {
+        const input = `
+        Alias: SOURCE = http://example.org/something
+
+        Invariant: rules-2
+        Severity: #error
+        Description: "This has a rule."
+        * source = SOURCE
+        `;
+
+        const result = importSingleText(input, 'InvariantRules.fsh');
+        expect(result.invariants.size).toBe(1);
+        const invariant = result.invariants.get('rules-2');
+        expect(invariant.rules).toHaveLength(1);
+        assertAssignmentRule(invariant.rules[0], 'source', 'http://example.org/something');
+      });
+    });
+
+    describe('#pathRule', () => {
+      it('should parse a pathRule', () => {
+        const input = `
+        Invariant: rules-3
+        Severity: #error
+        Description: "This has a rule."
+        * requirements
+        `;
+        const result = importSingleText(input, 'InvariantRules.fsh');
+        expect(result.invariants.size).toBe(1);
+        const invariant = result.invariants.get('rules-3');
+        expect(invariant.rules).toHaveLength(1);
```

Comment:
Well... It's complicated? I think I did it this way because `Invariant` is kind of like `Instance`, in that it's an instance of a constraint (and only allows assignments and inserts, just like instances). In theory, someone _could_ profile structure definition and slice things, etc -- but now that I think of it, I don't think we need to worry about that because -- (a) I don't think SUSHI would handle that right anyway, and (b) all the properties of `ElementDefinition.constraint` are `0..1` or `1..1`.

So... I think you're right. I think we can toss path rules at the import step. I'll do that when I get a chance.

Reply:
Applied in 593ad1a.